src/app/workflow/workflow-file-editor/workflow-file-editor.component.spec.ts

```diff
@@ -28,39 +28,41 @@ describe('WorkflowFileEditorComponent', () => {
   let component: WorkflowFileEditorComponent;
   let fixture: ComponentFixture<WorkflowFileEditorComponent>;
   class FileStubService {}
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      declarations: [
-        WorkflowFileEditorComponent,
-        CodeEditorListComponent,
-        CodeEditorComponent,
-        PublicFileDownloadPipe,
-        PrivateFileDownloadPipe,
-        PrivateFilePathPipe,
-      ],
-      imports: [
-        MatButtonModule,
-        MatTabsModule,
-        MatToolbarModule,
-        MatIconModule,
-        MatInputModule,
-        MatFormFieldModule,
-        MatTooltipModule,
-        ClipboardModule,
-        MatSnackBarModule,
-        MatCardModule,
-        BrowserAnimationsModule,
-        HttpClientModule,
-      ],
-      providers: [
-        { provide: HostedService, useClass: HostedStubService },
-        { provide: WorkflowService, useClass: WorkflowStubService },
-        { provide: WorkflowsService, useClass: WorkflowsStubService },
-        { provide: RefreshService, useClass: RefreshStubService },
-        { provide: FileService, useClass: FileStubService },
-      ],
-    }).compileComponents();
-  }));
```

Comment:
Looks like a whitespace change? 
Any idea why this occurred despite husky?

Reply:
I manually moved waitForAsync (probably by accident) to the next line. prettier considers this valid